src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-pricing-page/ubs-admin-tariffs-pricing-page.component.html

```diff
@@ -130,15 +130,17 @@ <h6>{{ 'ubs-tariffs.order_limits' | translate }}</h6>
                 (keypress)="checkOnNumber($event, minPriceOfOrder)"
                 id="from"
                 formControlName="minPriceOfOrder"
+                placeholder="0грн"
               />
-              <span class="line"></span>
+              <span class="line">-</span>
               <input
                 class="input"
                 type="number"
                 min="0"
                 (keypress)="checkOnNumber($event, maxPriceOfOrder)"
                 id="to"
                 formControlName="maxPriceOfOrder"
+                placeholder="0грн"
```

Comment:
the same)

Reply:
Thank you! Added translation of placeholder.